ionic2/src/app/app.component.ts

```diff
@@ -1,21 +1,29 @@
-import { Component } from '@angular/core';
-import { Platform } from 'ionic-angular';
+import { Component, ViewChild } from '@angular/core';
+import { Platform, Nav } from 'ionic-angular';
 import { StatusBar } from 'ionic-native';
 import 'rxjs/add/operator/map';
 
 import { MapPage } from '../pages/map/map.page';
+import { NotificationService } from '../services/notification.service';
 
 @Component({
   templateUrl: './app.html'
 })
 export class App {
   rootPage = MapPage;
 
-  constructor(private platform: Platform) {
+  @ViewChild('content') nav: Nav;
+
+  constructor(private platform: Platform, private notificationService: NotificationService) {
     platform.ready().then(() => {
-      // Okay, so the platform is ready and our plugins are available.
-      // Here you can do any higher level native things you might need.
-      StatusBar.styleDefault();
+      if (platform.is('cordova')) {
+        StatusBar.styleDefault();
+      }
+
+      if (platform.is('android')) {
```

Comment:
Since we are not planning on releasing an iOS or Windows phone application and I never tested it on anything other than Android I thought it's best to just support Android. If you want to test it on iOS and Windows phone feel free to do so :)
But thinking about this made me realize that there is probably something else wrong about this statement. `platform.is('android')` also returns true if the app is running inside a browser on an Android device. So this should be changed to `platform.is('android') && !platform.is('mobileweb')`.
https://ionicframework.com/docs/v2/api/platform/Platform/#is